src/components/organisms/Product/index.tsx

```diff
@@ -1,15 +1,24 @@
+import { Link } from 'react-router-dom'
+
 import Button from 'components/molecules/Button'
 
 import { ProductProps } from './Product.types'
 
-const Product = ({ name, image, price, currency }: ProductProps) => (
+const Product = ({ id, title, img, price, currency }: ProductProps) => (
   <div className='grid gap-2 rounded-lg bg-stone-200 p-5 shadow-xl'>
-    <img
-      src={image}
-      className='mb-4 aspect-[1/1] h-full w-full rounded-md object-fill object-center'
-    />
-    <h3 className='text-3xl'>{name}</h3>
-    <p className='mb-1 text-2xl font-light'>
+    <Link
+      className='mb-10'
+      to={`/products/${id}`}
+      onClick={() => window.scrollTo({ top: 0, behavior: 'smooth' })}
```

Comment:
just a quick note: window.scrollTo doesn't work on every device, you can keep it this way, but keep that information in mind